packages/connector-wechat-native/package.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "@logto/connector-wechat-native",
+    "version": "0.1.0",
+    "description": "WeChat native connector implementation.",
+    "main": "./lib/index.js",
+    "exports": "./lib/index.js",
+    "author": "Silverhand Inc. <contact@silverhand.io>",
+    "license": "MPL-2.0",
+    "files": [
+        "lib",
+        "docs"
+    ],
+    "private": false,
```

Comment:
nit: can remove this line.

Reply:
[LOG-2329](https://linear.app/silverhand/issue/LOG-2329/packagejson-private-default-value-can-be-removed)
Let's fix them all later.